src/app-admin.js

```diff
@@ -0,0 +1,491 @@
+import express from 'express';
+import * as uuid from 'uuid';
+import { errorMiddleware } from './util/middlewares.js';
+import validateUser from './util/validate-user.js';
+import getAccountDb, { isAdmin } from './account-db.js';
+import config from './load-config.js';
+import bodyParser from 'body-parser';
+
+let app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(errorMiddleware);
+
+export { app as handlers };
+
+export const TOKEN_EXPIRATION_NEVER = -1;
+
+const sendErrorResponse = (res, status, reason, details) => {
+  res.status(status).send({
+    status: 'error',
+    reason,
+    details,
+  });
+};
+
+const getUserByUsername = (userName) => {
+  return (
+    getAccountDb().first('SELECT id FROM users WHERE user_name = ?', [
+      userName,
+    ]) || {}
+  );
+};
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It feels like a lot of the user and other table SQL queries could be extracted to an `user-service.js` file with readable names. See the example of `secret-service.js` we already have.